dipy/segment/tests/test_mrf.py

```diff
@@ -28,32 +29,42 @@
 square[71:185, 71:185, :] = 2
 square[99:157, 99:157, :] = 3
 
-square_gauss = np.zeros((256, 256, 3)) + 0.001
-square_gauss = add_noise(square_gauss, 10000, 1, noise_type='gaussian')
-square_gauss[42:213, 42:213, :] = 1
-noise_1 = np.random.normal(1.001, 0.0001,
-                           size=square_gauss[42:213, 42:213, :].shape)
-square_gauss[42:213, 42:213, :] = square_gauss[42:213, 42:213, :] + noise_1
-square_gauss[71:185, 71:185, :] = 2
-noise_2 = np.random.normal(2.001, 0.0001,
-                           size=square_gauss[71:185, 71:185, :].shape)
-square_gauss[71:185, 71:185, :] = square_gauss[71:185, 71:185, :] + noise_2
-square_gauss[99:157, 99:157, :] = 3
-noise_3 = np.random.normal(3.001, 0.0001,
-                           size=square_gauss[99:157, 99:157, :].shape)
-square_gauss[99:157, 99:157, :] = square_gauss[99:157, 99:157, :] + noise_3
-
-square_1 = np.zeros((256, 256, 3)) + 0.001
-square_1 = add_noise(square_1, 10000, 1, noise_type='gaussian')
-temp_1 = np.random.randint(1, 21, size=(171, 171, 3))
-temp_1 = np.where(temp_1 < 20, 1, 3)
-square_1[42:213, 42:213, :] = temp_1
-temp_2 = np.random.randint(1, 21, size=(114, 114, 3))
-temp_2 = np.where(temp_2 < 19, 2, 1)
-square_1[71:185, 71:185, :] = temp_2
-temp_3 = np.random.randint(1, 21, size=(58, 58, 3))
-temp_3 = np.where(temp_3 < 20, 3, 1)
-square_1[99:157, 99:157, :] = temp_3
+
+@set_random_seed
+def create_square_uniform():
```

Comment:
Why is this change part of the commit?

Reply:
Because I had to add a decorator when creating the global variables. Adding the decorator to the test functions will be too late. Before, it was just defining them globally, which I do not think I can add a decorator to it.

Reply:
OK, then please explain in the commit message. Thanks Jong Sung.